README.md

```diff
@@ -740,79 +735,9 @@ d = encoder(wf)
 assert d == b[44:]
 
 sf_read = sf.read(BytesIO(b), dtype='int16')
-recode_read = list(decoder(d))
+recode_read = decoder(d)
 
 assert np.all(sf_read[0] == recode_read)
 assert np.all(recode_read == wf)
 assert np.all(sf_read[0] == wf)
 ```
-
```

Comment:
This part is outdated, because it is about a comparison with an implementation that has been modified since then, precisely as a result of this comparison!  
And indeed I don't think this should be in the README. OL put it in [this wiki](https://github.com/otosense/recode/wiki/Compare-performance-of-IterativeDecoder-and-ChunkedDecoder) too, which I will update.